### ISSUE TITLE
Add documentation for sbt dependencyTree plugin

### ISSUE THREAD

**Issue by manuzhang** [sbt/sbt#7485]
The only documentation I can find on the website is this [release note](https://www.scala-sbt.org/1.x/docs/sbt-1.4-Release-Notes.html#sbt-dependency-graph+is+in-sourced) and the archived [sbt-dependency-graph](https://github.com/sbt/sbt-dependency-graph). Is there an official doc for this plugin?

**Comment by mkurz** [comment]
Yeah, this was bothering me as well already. There is nothing more I think. I think the contents of the README should be integrated into the sbt docs.

**Comment by angabriel** [comment]
I am very interested in this as well. 

Using sbt 1.7.1 with addDependencyTreePlugin in build.sbt.

`sbt dependencyBrowseGraph` fails with `Not a valid key: dependencyBrowseGraph (similar: dependencyTreeModuleGraph0, dependencyTreeModuleGraphStore, dependencyTree)`. 

The mentioned readme on the old sbt-dependency-graph site does not help. Maybe i am missing something?

**Comment by mkurz** [comment]
> Maybe i am missing something?

`addDependencyTreePlugin` needs to go into `project/plugins.sbt` not `build.sbt`

**Comment by isaac-philip** [comment]
@Friendseeker is this still open ?

**Comment by Friendseeker** [comment]
> @Friendseeker is this still open ?

I don't have an ETA. I am having final exams coming soon (I am a student) but this is indeed something I active have in mind.

- I also won't mind other people building on top of my existing work & completing it, if you & others intend to take matters into own hand. Would really appreciate help from your guys as I need to deal with real life commitments.

**Comment by isaac-philip** [comment]
@Friendseeker was only asking if the issue is open for contribution 😃  , can take a look, thanks.

**Comment by Friendseeker** [comment]
> @Friendseeker was only asking if the issue is open for contribution 😃  , can take a look, thanks.

Right my fault. It is a different ticket! I thought the ticket was referring to dependency tree plugin refactor I was working on.

Yes would definitely welcome contributions!

Apologies for any confusion.

**Comment by Dairus01** [comment]
Hi @Friendseeker @eed3si9n, if no one is working on this issue, can I be assigned to it?

**Comment by eed3si9n** [comment]
Sure. The documentation should be somewhere under https://github.com/sbt/website/tree/develop/src/reference/reference. Note this is outside of sbt/sbt.
